README.md

```diff
@@ -87,17 +87,8 @@ initialize and recursively update submodules.  You will find it most
 convenient to setup public-key login for GitHub before you perform
 this step.
 
-Now, you may use either `stack` or `cabal new-build` to compile the
-libraries, as you prefer.
-
-```
-ls stack-ghc-*.yaml
-# Choose the GHC version you prefer
-ln -s stack-ghc-<version>.yaml stack.yaml
-./scripts/build-sandbox.sh
-stack setup
-stack build
-```
+Crucible can be built with the `cabal` tool (although it is not compatible with
+the old/v1 cabal version).
 
 ```
 ./scripts/build-sandbox.sh
```

Comment:
As a side note (not necessarily something that you need to do in this PR), I wonder if this script is even worth having. All it does currently is:

https://github.com/GaloisInc/crucible/blob/2cf85d801d806a5f62cfc19403e041169be66cb7/scripts/build-sandbox.sh#L1C1-L3

We could just as well inline this into the `README` instructions. Moreover, I'm not sure how I feel about the `--recursive` part, since that will likely cause submodules to clone _other_ submodules that aren't necessarily for the `crucible` build.